helm_deploy/values-prod.yaml

```diff
@@ -6,6 +6,11 @@ generic-service:
   ingress:
     host: frontend-components.hmpps.service.justice.gov.uk
 
+  strategy:
```

Comment:
I've just checked https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec and can see that you don't need `type: RollingUpdate` since that's the default but is it worth explicitly setting to save us a job if they for some reason change the default in a breaking change release later?